integration-tests/src/mock_network/setup.rs

```diff
@@ -313,6 +313,15 @@ pub fn setup_mock_network(
                 target_height,
             )
         });
+    #[cfg(not(feature = "test_features"))]
```

Comment:
I'm surprised that rpc server depends on test_features -- can you add a comment explaining why? (or consider simply enabling it always)

Reply:
yeah it's because with test_features, rpc server requires PeerManagerActor, which is not available here, so I simply just require no test feature here.